src/core/Radio/index.tsx

```diff
@@ -13,8 +13,9 @@ export interface RadioProps
  * @see https://v4.mui.com/components/radio-buttons/
  */
 const RadioButton = (props: RadioProps): JSX.Element => {
+  const { stage } = props;
```

Comment:
this was a linter warning that I fixed while trying to resolve other issues.  It's not strictly related, but I figured it doesn't hurt to get rid of a linter warning.